### PR TITLE
interfaces/udisks2: Allow access to the login manager via dbus

### PR DESCRIPTION
**Pull request by alexmurray**
References:
https://forum.snapcraft.io/t/udisks2-interface-extend-permissions/24073

### CODE REVIEW

File: interfaces/builtin/udisks2.go [hunk 1]
```diff
@@ -64,6 +64,20 @@ dbus (send)
...
     peer=(label=unconfined),
 
+# Allow accessing logind services to reinitialise devices on resume
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/login1
+    interface=org.freedesktop.login1.Manager
+    member={PrepareForSleep}
+    peer=(label=unconfined),
+# do not use peer=(label=unconfined) here since this is DBus activated
```
Comment (mvo5) [inline interfaces/builtin/udisks2.go:74]: Thanks for this comment

**Review by mvo5 — APPROVED** [review]
Thank you, looks good!

**Review by anonymouse64 — APPROVED** [review]
+1 thanks for looking into this

**Comment by mvo5** [comment]
The error in the log looks real:
```
2021-05-03T09:49:53.6389061Z 2021-05-03 09:49:53 Restoring google:ubuntu-core-16-64:tests/main/interfaces-broadcom-asic-control (may030938-510221)...
2021-05-03T09:49:55.1709349Z ##[error]2021-05-03 09:49:55 Error executing google:ubuntu-core-16-64:tests/main/interfaces-many-snap-provided (may030938-510248) : 
2021-05-03T09:49:55.1733146Z -----
2021-05-03T09:49:55.1733897Z + . /home/gopath/src/github.com/snapcore/snapd/tests/lib/dirs.sh
2021-05-03T09:49:55.1734923Z ++ export SNAP_MOUNT_DIR=/snap
2021-05-03T09:49:55.1735436Z ++ SNAP_MOUNT_DIR=/snap
2021-05-03T09:49:55.1736025Z ++ export LIBEXECDIR=/usr/lib
2021-05-03T09:49:55.1754007Z ++ LIBEXECDIR=/usr/lib
2021-05-03T09:49:55.1754615Z ++ export MEDIA_DIR=/media
2021-05-03T09:49:55.1755092Z ++ MEDIA_DIR=/media
2021-05-03T09:49:55.1755572Z ++ case "$SPREAD_SYSTEM" in
2021-05-03T09:49:55.1757284Z + PROVIDER_SNAP=test-snapd-policy-app-provider-classic
2021-05-03T09:49:55.1758240Z + os.query is-core
2021-05-03T09:49:55.1759144Z + PROVIDER_SNAP=test-snapd-policy-app-provider-core
2021-05-03T09:49:55.1760101Z + echo 'Given a snap is installed'
2021-05-03T09:49:55.1760775Z Given a snap is installed
2021-05-03T09:49:55.1762347Z + /home/gopath/src/github.com/snapcore/snapd/tests/lib/tools/snaps-state install-local test-snapd-policy-app-provider-core
2021-05-03T09:49:55.1763826Z error: cannot perform the following tasks:
2021-05-03T09:49:55.1765977Z - Setup snap "test-snapd-policy-app-provider-core" (unset) security profiles (cannot setup profiles for snap "test-snapd-policy-app-provider-core": cannot load apparmor profiles: exit status 1
2021-05-03T09:49:55.1768339Z apparmor_parser output:
2021-05-03T09:49:55.1769057Z apparmor_parser: Regex grouping error: Invalid number of items between {}
2021-05-03T09:49:55.1770292Z apparmor_parser: Unable to parse input line '{PrepareForSleep}'
2021-05-03T09:49:55.1772442Z ERROR processing policydb rules for profile snap.test-snapd-policy-app-provider-core.udisks2, failed to load
2021-05-03T09:49:55.1773868Z )
2021-05-03T09:49:55.1774388Z -----
```
Maybe on xenial there is something missing?

**Comment by alexmurray** [comment]
Thanks for the heads up @mvo - should be fixed by 20d105e but let's wait for spread to finish to make sure.

**Comment by mvo5** [comment]
Thanks, that update worked. The current spread failures are unrelated, the day is having a bad day and the "snap advertise" command gets 403 from the respective store API.

**Comment by alexmurray** [comment]
Thanks @mvo5 :)

**Pull request merged**